content/developers/index.md

```diff
@@ -1,7 +1,148 @@
 ---
 title: Quick start
-description: Our mission as Mysterium Node runners
+description: How to setup your environment
 ---
 
 ## Quick start for developers
-...
+
+Setting up your environment for building, running and testing Mysterium node doesnt have to be hard.
+The minimum required software to set up is:
+
+| Software           | Minimum Version |
+|--------------------|-----------------|
+| Go                 | 1.14.0+         |
+| Docker             | 19.0+           |
+| Docker Compose     | 1.21.0+         |
+| Mage (Recommended) | 1.10+           |
+
+
+<details><summary>General (Mage)</summary>
+<p>
+
+[Mage](https://github.com/magefile/mage) is a simple make-like Go tool we use to help us develop our software.
+Before installing Mage, make sure you've completed the below steps for system setup
+as it requires you to have `golang` installed on your system.
+
+To install simply execute these commands:
+
+```sh
+git clone https://github.com/magefile/mage
+cd mage
+go run bootstrap.go
+```
+
+To test if it's working, try executing `mage` in any directory.
```

Comment:
You're asking the reader to test what's bound to fail. Instructions should follow a happy path, however if there are edge cases we need to cover we should describe them as such. We should also give example instructions relevant to our project i.e. `mage ci check` telling the reader what this command does and when one would use it, not just generic instruction on the tool. Worth remembering that there will be people who are new to these tools and the best way to learn them is by real world examples:)